tests/test_runs_gc_performance.py

```
@@ -0,0 +1,55 @@
+    """Verify that discover_all_runs skips size computation when compute_size=False."""
+
+    # Mock directories
+    mock_runs_dir = MagicMock(spec=Path)
+    mock_examples_dir = MagicMock(spec=Path)
+
+    # Setup mock runs
+    # Run 1: Active run with meta.json
+    run1 = MagicMock(spec=Path)
+    run1.name = "run-1"
+    run1.is_dir.return_value = True
+    run1.stat.return_value.st_mtime = datetime.datetime.now().timestamp()
```

Ruff (0.15.2)

[warning] `datetime.datetime.now()` called without a `tz` argument

(DTZ005)